hypershift-operator/controllers/hostedcluster/assets/autoscaler/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  labels:
+    app: cluster-autoscaler
+spec:
+  selector:
+    matchLabels:
+      app: cluster-autoscaler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cluster-autoscaler
+    spec:
+      containers:
+        - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.20.0
```

Comment:
we will need a follow-up to pull this image from our release payload.

i think this is the first component where its not clear if we should version this with hypershift release or ocp release.

thoughts?